libacm/CMakeLists.txt

```diff
@@ -9,3 +12,9 @@ set(LIB_SRC
   libacm.h)
 
 add_library(libacm STATIC ${HEADERS} ${CPPS} ${LIB_SRC})
+
+target_include_directories(libacm PUBLIC
+  $<BUILD_INTERFACE:
+    ${PROJECT_SOURCE_DIR}/libacm
```

Comment:
ideally this module should only expose `adecode.h/aencode.h` and not `libacm.h` from upstream libacm, but that'll require moving it to a third party (idea for a future improvement)